specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-02-15-preview/cosmos-db.json

```diff
@@ -8632,6 +8636,55 @@
         }
       }
     },
+    "ClientEncryptionPolicy": {
+      "type": "object",
+      "description": "Cosmos DB client encryption policy.",
+      "properties": {
+        "includedPaths": {
+          "description": "Paths of the item that need encryption along with path-specific settings.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClientEncryptionIncludedPath"
+          }
+        },
+        "policyFormatVersion": {
```

Comment:
Track 1 for some reason end ups showing this parameter in the constructor and is not taking a default value. Checked with the .NET swagger team. They say track 2 they support it. When i use Default: 1 it doesnt seem to take it after making this field ReadOnly. But the problem is we cannot set it in PSClientEncryptionPolicy since it will readonly.